packages/sdks/src/components/render-block/render-block.lite.tsx

```diff
@@ -91,20 +92,29 @@ export default function RenderBlock(props: RenderBlockProps) {
   });
 
   return (
-    <>
+    <Show
+      when={!state.componentInfo?.noWrap}
+      else={
+        <state.componentRef
+          attributes={state.propertiesAndActions}
+          {...state.componentOptions}
+          builderBlock={state.useBlock}
+          style={state.css}
+        >
+          <For each={state.children}>
+            {(child) => <RenderBlock key={child.id} block={child} />}
+          </For>
+        </state.componentRef>
+      }
+    >
       <Show
-        when={!state.componentInfo?.noWrap}
+        when={!isEmptyHtmlElement(state.tagName)}
```

Comment:
No need to check for `noWrap` anymore?

Reply:
What I actually did is _add_ another nested `Show` statement, not alter the existing one. If you look at the diff with whitespace ignored: https://github.com/BuilderIO/builder/pull/1032/files/c6dbd9537c5e6e42a3a3c8cdd180686b20adfef?w=1

<img width="1578" alt="image" src="https://user-images.githubusercontent.com/1393142/170587815-2983f201-044b-47c5-8953-0fb61e5a1f3c.png">


it should be easier to parse and see 🤞🏽

Reply:
ah yes! i missed that